### PR TITLE
Minor update styles packages manager

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Updated styles for package manager

### Why is it needed?
Improves the visual perception of text information

Before:
Снимок экрана 2021-03-24 в 21 06 16

After:
Снимок экрана 2021-03-24 в 21 09 52

### How to test
Rebuild assets

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/15564

### CODE REVIEW

**Review by alroniks — APPROVED** [review]

File: _build/templates/default/sass/_package-management.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  ul {
+    margin-bottom: 1rem;
+  }
+}
+
+#modx-package-browser-details-main {
+  p {
+    margin-bottom: 0.3rem;
+    color: #808080;
```
Comment (rthrash) [inline _build/templates/default/sass/_package-management.scss:23]: For normal text, this doesn't pass accessibility, see: https://webaim.org/resources/contrastchecker/?fcolor=808080&bcolor=FFFFFF

To get it to pass would require going to #595959 for the text, and that would also probably need a darker color for the headline for visual balance.

**Review by rthrash — CHANGES_REQUESTED** [review]
I think we need to look at slightly more contrast. I love the way it looks, so great job, but some with lesser vision may struggle to read it.

**Review by Ruslan-Aleev — APPROVED** [review]

**Comment by theboxer** [comment]
@Ibochkarev any plans on updating this PR with the requested changes?

**Comment by Ibochkarev** [comment]
> @Ibochkarev any plans on updating this PR with the requested changes?

I can offer to accept it as it is (since the branch was deleted from me and I cannot make changes), and then make these changes in the next request.

**Comment by theboxer** [comment]
@Ibochkarev you can either restore the branch following this tutorial: https://garrytrinder.github.io/2020/05/recover-pr-after-accidentally-deleting-source-branch
or if you have [github cli](https://cli.github.com/) tool (highly recommended), you can checkout to the PR branch by `gh pr checkout 15588` and try to push it to your repo
or cherry pick commits from this PR to a new branch and create a new PR
or create a new branch and copy & paste changes

I don't think that merging unfinished PR and then waiting for a new one to finish the work is a good approach.

**Comment by Ibochkarev** [comment]
Closed in favor of #15862

**Comment by Ibochkarev** [comment]
> @Ibochkarev you can either restore the branch following this tutorial: https://garrytrinder.github.io/2020/05/recover-pr-after-accidentally-deleting-source-branch or if you have [github cli](https://cli.github.com/) tool (highly recommended), you can checkout to the PR branch by `gh pr checkout 15588` and try to push it to your repo or cherry pick commits from this PR to a new branch and create a new PR or create a new branch and copy & paste changes
> 
> I don't think that merging unfinished PR and then waiting for a new one to finish the work is a good approach.

@theboxer 

``` shell
➜  revolution git:(3.x) gh pr checkout 15588
GraphQL error: Could not resolve to a PullRequest with the number of 15588.
```

``` shell
➜  revolution git:(3.x) git checkout -b package-browser e06e917f429ace4442d24a042b1301fed5d4fa3f
fatal: there is no such link in the village: e06e917f429ace4442d24a042b1301fed5d4fa3f
```

**Pull request closed without merging**